core/src/main/java/com/netease/arctic/AlreadyExistsException.java

```diff
@@ -22,4 +22,24 @@ public class AlreadyExistsException extends RuntimeException {
   public AlreadyExistsException(String message) {
     super(message);
   }
+
+  public AlreadyExistsException() {
+    super();
```

Comment:
I cannot find why we need this constructor with no arguments, and it seems to be not used.